core/shared/src/main/scala/zio/prelude/data/Optional.scala

```diff
@@ -50,7 +50,7 @@ sealed trait Optional[+A] { self =>
       case Optional.Absent       => ifEmpty
     }
 
-  final def flatten[B](implicit ev: A <:< Option[B]): Option[B] =
+  final def flatten[B](implicit ev: A <:< Optional[B]): Optional[B] =
```

Comment:
I didn't understand why the evidence what proving that `A` was an `Option[B]` nor why this method was returning an `Option[B]` and not an `Optional[B]`.
Probably a copy/paste mistake